examples/gcp_bigquery_big_lake_table/main.tf

```diff
@@ -141,6 +146,9 @@ resource "google_storage_bucket_object" "dummy_parquet_file" {
 resource "google_storage_bucket_object" "dummy_csv_file" {
   bucket       = google_storage_bucket.big_lake_data_source.id
   name         = "csv/year=2023/month=12/day=18/test.csv"
-  content      = "hello"
+  content      = <<-EOT
+  dummy1|dummy2
```

Comment:
Please rename dummy1 and dummy2 to header1 and header2 or Column1Name and Column2Name.